src/importexport/ove/internal/ove.cpp

```diff
@@ -4459,6 +4459,8 @@ bool OvscParse::parse()
     }
     m_ove->setShowColor(placeHolder.toBoolean());
 
+    m_handle = nullptr;
+
```

Comment:
Here you left out the other 3 locations

Reply:
Huh? No idea what a _a RAII-based solution_ might be. What I know is that this change is is not enough, jumping (way) too short.

Reply:
But if you find a simpler one that you just did, go ahead.

Reply:
My terminology might be strange, but I've written up my idea and pushed it to here.